maat-court-data-api/src/main/java/gov/uk/courtdata/model/assessment/PassportAssessment.java

```diff
@@ -0,0 +1,55 @@
+package gov.uk.courtdata.model.assessment;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+import java.time.LocalDateTime;
+
+@Data
+@SuperBuilder
+@AllArgsConstructor
+@NoArgsConstructor
+public class PassportAssessment {
+    private Integer repId;
+    private String nworCode;
+    private Integer cmuId;
+    private LocalDateTime assessmentDate;
+    private String partnerBenefitClaimed;
+    private String partnerFirstName;
+    private String partnerSurname;
+    private String partnerOtherNames;
+    private String partnerNiNumber;
+    private LocalDateTime partnerDob;
+    private String incomeSupport;
+    private String jobSeekers;
+    private String statePensionCredit;
+    private String under18FullEducation;
+    private String under16;
+    private String pcobConfirmation;
+    private String result;
+    private String dwpResult;
+    private String passportNote;
+    private String between16And17;
+    private String under18HeardInYouthCourt;
+    private String under18HeardInMagsCourt;
+    private LocalDateTime lastSignOnDate;
+    private String esa;
+    private String pastStatus;
+    @Builder.Default
+    private String replaced = "N";
```

Comment:
This is already handled by the entity, it doesn't need to be part of the schema.

Reply:
This is the parent class for create/update objects which only have fields specific to them.

Reply:
Yes, but the only time replaced is ever set to 'Y' is when a new one is created with a matching RepID, which is already being handled. The entity then sets replaced to 'N' by default, so I don't think we don't need to expose the field.

`@Builder.Default`
`@Column(name = "REPLACED")`
`private String replaced = "N";`